apps/web/src/modules/create-proposal/components/TransactionForm/Migration/MigrationFormC2.tsx

```diff
@@ -0,0 +1,125 @@
+import { Box, Button, Flex, Text } from '@zoralabs/zord'
+import { Form, Formik, FormikHelpers } from 'formik'
+import { useCallback, useState } from 'react'
+
+import SmartInput from 'src/components/Fields/SmartInput'
+import {
+  defaultHelperTextStyle,
+  defaultInputLabelStyle,
+} from 'src/components/Fields/styles.css'
+import { TEXT } from 'src/components/Fields/types'
+
+import { DropdownSelect } from '../../DropdownSelect'
+import MigrationFormC2Schema, { MigrationFormC2Values } from './MigrationFormC2.schema'
+
+export interface MigrationFormC2Props {
+  onSubmit?: (
+    values: MigrationFormC2Values,
+    actions: FormikHelpers<MigrationFormC2Values>
+  ) => void
+  disabled?: boolean
+}
+
+export type ChainType = 'BASE' | 'ZORA' | 'OP'
+
+const chainOptions = [
+  { label: 'Base', value: 'BASE' },
```

Comment:
value for this should be chainId so we can easily get the chain object / use any of the constants from the selection